project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.16
```

Comment:
copyright?

Reply:
there is no copyright in the build.properties files for other services